src/collections/vec.c

```diff
@@ -79,11 +79,13 @@ void _z_vec_append(_z_vec_t *v, void *e)
     if (v->len == v->capacity)
     {
         // Allocate a new vector
-        size_t _capacity = 2 * v->capacity;
+        size_t _capacity = (v->capacity << 1) | 0x01;
```

Comment:
This should be:

`size_t _capacity = 2 * v->capacity;
if (_capacity == 0) {
    1
}`

Reply:
done